### ISSUE TITLE
Warning in colab

### ISSUE THREAD

**Issue by domoritz** [flekschas/jupyter-scatter#93]
When I use the widget in colab, I get this warning

```
/usr/local/lib/python3.10/dist-packages/anywidget/_util.py:201: UserWarning: anywidget: Live-reloading feature is disabled. To enable, please install the 'watchfiles' package.
  start_thread=_should_start_thread(maybe_path),
```

Code

```
import jscatter
import numpy as np

x = np.random.rand(500)
y = np.random.rand(500)

jscatter.plot(x, y)
```

**Comment by flekschas** [comment]
Oh, I've seen this warning before. It's related to anywidget and shouldn't impact the widget at all. I actually wonder why this warning is shown as it should really only appear in development mode. @manzt Any idea if this is expected?

**Comment by manzt** [comment]
hey! We turn on auto-reloading when packages is outside of `site-packages`: 

https://github.com/manzt/anywidget/blob/65c03e0887683d760bb05de29df92f29dc8ae57e/anywidget/_util.py#L169

I'd be happy to also ignore `dist-packages`

**Comment by flekschas** [comment]
@manzt it seems like for Colab, the packages are always located outside of `site-packages` which generates the warning.

<img width="2249" alt="Screenshot 2023-09-15 at 7 23 58 PM" src="https://github.com/flekschas/jupyter-scatter/assets/932103/400f91ef-41b1-4f43-91ed-91ad822e1cb0">

Not sure it's possible to detect in which environment a widget is installed but if it is maybe the warning can be limited to Jupyter environments where the package is outside the `site-packages` folder?

**Comment by manzt** [comment]
should be fixed in anywidget v0.6.4

**Comment by flekschas** [comment]
Awesome thanks!
